server/scripts/participant-stats-rejected.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable no-console */
+require('dotenv').config({ path: '../.env' });
+const csv = require('fast-csv');
+const path = require('path');
+const { writeFileSync } = require('fs');
+const { dbClient } = require('../db/db.js');
+const { getRejectedParticipantsReport } = require('../services/reporting');
+
+(async () => {
+  if (require.main === module) {
+    try {
+      await dbClient.connect();
+
+      const results = await getRejectedParticipantsReport();
+
+      console.log(results);
```

Comment:
probably want to remove this, there should be no PII in the report but it will have HUGE output on prod